website/package.json

```diff
@@ -42,9 +42,9 @@
     ]
   },
   "dependencies": {
-    "@docusaurus/core": "^2.0.0-beta.14",
-    "@docusaurus/plugin-pwa": "^2.0.0-beta.14",
-    "@docusaurus/preset-classic": "^2.0.0-beta.14",
+    "@docusaurus/core": "0.0.0-4738",
+    "@docusaurus/plugin-pwa": "0.0.0-4738",
+    "@docusaurus/preset-classic": "0.0.0-4738",
```

Comment:
Can we wait until next `beta` is released? The class name issue doesn't seems that crucial to stick to nightlines. 😅 

Saying that, I was planning to upgrade to latest beta this weekend, so if changing that to `beta.17` also helps that would be fine.